packages/amplify_datastore/android/build.gradle

```diff
@@ -57,8 +57,8 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation "com.amplifyframework:aws-datastore:1.6.10"
-    implementation "com.amplifyframework:aws-api-appsync:1.6.10"
+    implementation "com.amplifyframework:aws-datastore:1.16.13"
+    implementation "com.amplifyframework:aws-api-appsync:1.16.13"
```

Comment:
Should probably be removed in the future now that we have the Amplify Flutter API plugin.